From c18b050844fd5a55eb8640891caed5b9f33b891d Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Tue, 19 Aug 2014 19:30:58 -0400
Subject: [scsi] qla4xxx: 5.04.00.04.07.01-k0

Message-id: <1408476658-5056-21-git-send-email-cdupuis@redhat.com>
Patchwork-id: 88351
O-Subject: [PATCH RHEL 7.1 e-stor 20/20] qla4xxx: 5.04.00.04.07.01-k0
Bugzilla: 1089349
RH-Acked-by: Rob Evers <revers@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Vikas Chaudhary <vikas.chaudhary@qlogic.com>

RHEL 7 specific.

Signed-off-by: Vikas Chaudhary <vikas.chaudhary@qlogic.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/qla4xxx/ql4_version.h b/drivers/scsi/qla4xxx/ql4_version.h
index 8746355..dff31eb 100644
--- a/drivers/scsi/qla4xxx/ql4_version.h
+++ b/drivers/scsi/qla4xxx/ql4_version.h
@@ -5,4 +5,4 @@
  * See LICENSE.qla4xxx for copyright and licensing details.
  */
 
-#define QLA4XXX_DRIVER_VERSION "5.04.00.04.07.00-k0"
+#define QLA4XXX_DRIVER_VERSION "5.04.00.04.07.01-k0"
-- 
1.7.1